### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by HarrisonMS**

### CODE REVIEW

File: api/server.spec.js [hunk 1]
```diff
@@ -18,17 +18,17 @@ describe("server", () => {
...
+            expect(Array.isArray(response.body)).toBe(false);
           })
       );
     });
 
     it("should return { api: 'up' }", () => {
       return supertest(server)
         .get("/")
-        .then(response => {
+        .then((response) => {
```
Comment (bradzickafoose) [inline api/server.spec.js:31]: Is this prettier's handiwork or are you going back and adding parenthesis around the argument?

Comment (bradzickafoose) [inline api/server.spec.js:31]: Is this Prettier's handiwork or are you going back and adding parentheses around your arguments?

**Review by bradzickafoose — APPROVED** [review]
🔥

File: api/server.spec.js (hunk 1 above)
Comment (HarrisonMS) [reply api/server.spec.js:31]: Yea that'd be prettier

**Pull request merged**